engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -362,6 +362,72 @@ enum SemanticsRole {
 
   /// The main display for a tab.
   tabPanel,
+
+  /// An input field for users to enter search terms.
+  searchBox,
+
+  /// A control that used for dragging across content.
+  ///
+  /// e.g, the drag handle of [ReorderableList].
+  dragHandle,
+
+  /// A control to cycle through content on tap.
+  ///
+  /// e.g, the next and previous month button of a [CalendarDatePicker].
+  spinButton,
+
+  /// A input field with a dropdown list box attached.
+  ///
+  /// e.g, a [DropDownMenu]
+  comboBox,
+
+  /// Contains a list of [menu].
```

Comment:
nit: just like you did `[menuItem]s` below, do `[menu]s` here too?

Reply:
menu is the correct word, a menu is one button that will open dropdown on click. a menu bar will contains multiple menus

Reply:
Sorry, that's not what I meant. I just want to replace `list of [menu]` with `list of [menu]s`.

Reply:
oh sorry I misread fixing now